packages/api-client/docs/AssessmentApi.md

```
@@ -1,6 +1,6 @@
 # AssessmentApi
 
-All URIs are relative to *https://raw.githubusercontent.com*
+All URIs are relative to *http://localhost*
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)